projects/uitdatabank/docs/entry-api/project-flow.md

```diff
@@ -0,0 +1,60 @@
+---
+stoplight-id: 73ef6e15ee3ec
+---
+
+# Project flow from test to production
+
+For the integration with the UiTdatabank Entry API, there is a flow in which you first obtain access to our test environment. 
+Access to production environment is given after your integration on test environment has been successful validated by publiq vzw.
+
+On this page we describe how this flow works and how you can ensure that you gain access to the production environment as quickly as possible!
+
+> Make sure you read this guide before you start developing on our test environment!
+
+## Step 1: start on test environment
```

Comment:
```suggestion
## Step 1: start on the test environment
```

Hmm again a missing "the" I think, or is this purposeful without "the"?

Reply:
The "the" was left out on purpose here. Not sure why, maybe because there is no such thing as "the" test environment (there are many test environments, e.g. for authentication, UiTdatabank, etc.)

Reply:
Hmm I feel like we say "the" test environment a lot? An environment is not limited to 1 app, it can be a collection of apps. So the test tenant of auth0, test server of UiTdatabank Entry API / Search API, test server of UiTPAS, etc are all together "the" test environment in my view.